index.css

```diff
@@ -97,11 +102,15 @@ input:focus {
 
 input.good {
     opacity: 1;
+    rotate: 2deg;
+    border-radius: 25px;
 }
 input.bad {
     background-color: black!important;
     opacity: 0.7;
     color: #fff!important;
+    rotate: 2deg;
```

Comment:
I think it would be really fun to make this a random number between -3 and +3. We can do this with JS